.taprc

```diff
@@ -1,4 +1,5 @@
 ts: false
+check-coverage: false
```

Comment:
took it from pino, otherwise tests will fail if not reaches 100% coverage. https://github.com/pinojs/pino/blob/f8696ab5fb0e5a819b6255af420b29fb075cbdee/.taprc#L3

Reply:
Please add the istanbul ignores. Let's not lower the code coverage here.